src/NuGet.Core/NuGet.Commands/RestoreCommand/DependencyGraphResolver.cs

```diff
@@ -1411,11 +1414,12 @@ .. chosenResolvedItem.Suppressions
 
         private static bool IsNewerThanNET10(NuGetFramework frameworkName)
         {
-            if (frameworkName.Framework == FrameworkConstants.FrameworkIdentifiers.NetCoreApp)
-            {
-                return frameworkName.Version.Major >= 10;
-            }
-            return false;
+            return true;
+            //if (frameworkName.Framework == FrameworkConstants.FrameworkIdentifiers.NetCoreApp)
+            //{
+            //    return frameworkName.Version.Major >= 10;
+            //}
+            //return false;
```

Comment:
Can we remove this?

Reply:
I actually should've reverted that.
None of tests broke though, so I'm obviously missing some coverage for that. 

Looking into it.

Reply:
Fixed now.